src/chrome-storage.store.ts

```diff
@@ -0,0 +1,31 @@
+import { AbstractStringValuedAsyncStore } from './abstract-string-valued.store';
+
+export class ChromeStorageAsyncStore<T> extends AbstractStringValuedAsyncStore<T> {
```

Comment:
With the rename and big changes, GitHub diff thinks this is effectively a new file 🤷‍♂️ 

Compare to the "Deleted" file below